ensime_shared/ensime.py

```diff
@@ -156,21 +155,23 @@ def queue_poll(self, sleep_t=0.5):
         Value of sleep is low to improve responsiveness.
         """
         connection_alive = True
+
         while self.running:
             if self.ws:
-                def logger_and_close(m):
-                    self.log("Websocket exception: {}".format(m))
+                def logger_and_close(msg):
+                    self.log("Websocket exception: {}".format(msg))
                     if not self.running:
                         # Tear down has been invoked
                         # Prepare to exit the program
-                        connection_alive = False
+                        connection_alive = False  # noqa: F841
```

Comment:
Linter understandably thinks this is an unused local variable—this *is* pretty tricky code, in part this is a symptom of `with catch()` encouraging this deeply nested local function…